middlewares/Authentication.js

```diff
@@ -88,4 +88,46 @@ export default class Authentication {
 
     return next();
   }
+
+  /**
+   *Extract request user id if token is present
+   *
+   * @static
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {function} next
+   *
+   * @returns {function} next function
+   *
+   * @memberof Authentication
+   */
+  static async fetchRequester(req, res, next) {
```

Comment:
Big props to you for this abstraction!! 👍🏽

Reply:
Thank you.